doc/workshop/generation/dtm/gdal.json

```diff
@@ -1,6 +1,6 @@
 {
     "pipeline": [
-        "./exercises/analysis/ground/denoised-ground-only.copc.laz",
```

Comment:
Is there no `copc` file with the exercises to work with?

Reply:
Just looked, think I might have gotten confused there from an old version of the ground exercise that output laz only. Newest version makes copc